src/IP.php

```diff
@@ -62,12 +67,127 @@ abstract class IP
     /**
      * @var bool
      */
-    protected $is_private;
+    private $is_private;
```

Comment:
what is the reason for making this private?

Reply:
Because the `isPrivate()` method was moved inside the base class (no longer abstract), this property does not need to be accessed by any object that extends it - it is an internal reference.